admin-dev/themes/new-theme/js/translation-page/messages-search.js

```diff
@@ -0,0 +1,27 @@
+import Jets from 'jets/jets';
+import $ from 'jquery';
+
+export default function() {
+    $(() => {
+        $('.search-translation form').submit(function () {
+            $('#jetsContent form').addClass('hide');
```

Comment:
When inspecting the dom, I find it more obvious with classes which element should be visible or not without having to inspect element by element. Moreover with classes one can tell how many items are hidden in console using the appropiate class selector. It makes testing easier.